### DISCUSSION TITLE
Is there an graphql api can query endpoint top N percentile(p99,p95,p90,p75,p50) latency?

### DISCUSSION THREAD

**Discussion by PeterLiuWT00376** [apache/skywalking#10091] [Q&A] [answered]
I have tried below method, but it cannot work.
sortMetrics(condition: TopNCondition!, duration: Duration!): [SelectedRecord!]!
{
    sortMetrics: sortMetrics(
    condition: {
        name:"endpoint_percentile",
        normal: true,
        order:DES,
        parentService:"pe::pe-swap-biz",
        scope: Endpoint,
        topN: 10
    }, duration:{
            start:"2022-12-02 1435",
            end:"2022-12-02 1505",
            step:MINUTE
        })
    {   
        name
        id
        value
        refId
    }    
}

**Comment by wu-sheng** [comment, marked as answer]
No, you can't. I think this had been discussed. Search before submit a new one.
